salt/modules/win_lgpo.py

```diff
@@ -2588,7 +2588,9 @@ def _sidConversion(cls, val, **kwargs):
                     userSid = '{1}\\{0}'.format(userSid[0], userSid[1])
                 else:
                     userSid = '{0}'.format(userSid[0])
+            # TODO: This needs to be more specific
```

Comment:
Is there a reason we're adding `TODO`s here instead of just making these all more specific now?

Reply:
Because I don't know what the actual failures may be.

Reply:
If we had some automated tests for this then I think we could remove the blanket exception handling all together.

Reply:
I guess I don't really get what this is adding. We're just logging a line that we hit the exception, and in many cases handling and then moving on. I don't see how that is very useful to the end user.

Reply:
`log.exception` gives us the actual location where the exception is occurring. Otherwise, the rest of the code catches the exceptions as they bubble up and it makes it hard to find where the actual issue is.

Reply:
OK, thanks for the explanation!